picard/webservice/__init__.py

```diff
@@ -572,6 +577,22 @@ def get_response_parser(cls, response_type):
         else:
             raise UnknownResponseParserError(response_type)
 
+    @staticmethod
+    def get_last_last_raw_reply():
+        """ get last raw reply - the http body
+
+        This class passes the http body to a parsing function, and returns the result of that as the document.
+        Note that reply.readAll() is only able to be called once, as it consumes the actual http body.
+        Therefore the unparsed reply is not available to anything else.
+        Now we pre-saved the the http body and make it available via this getter.
+        Ideally this would not be static.  However, Webservice class is abstracted with layers of classes that make it unavailable from any class that would want to get the raw reply.
+        So making it static allows us to expose this value without re-architecting the whole program.
+        This should be re-factored in the future.
+        WARNING: This is not thread-safe, so if multiple threads are making Webservice calls concurrently, they could get the wrong results here.
+        But given we have the rate-limiter, we should be fine until this is refactored.
+        """
+        return last_raw_reply
```

Comment:
This only works by accident - `WebService._handle_reply` assigns to the _global_ variable `last_raw_reply`, not the one member of the `WebService` class. This should be `return WebService.last_raw_reply` (likewise for the assignments to it).

Reply:
The Match dialog is intended to show results of matching process.  So in middle pane, you must first either "Scan" which uses AcoustId, or "Lookup" which uses MusicBrainz.  Then the right click menu option is available on right pane to show all the results of that match, and allows you to choose a different result if desired.

Good catch on the last_raw_reply, I've fixed it.